src/main/java/org/nuxeo/ecm/csv/CSVImporter.java

```diff
@@ -43,4 +46,18 @@ String launchImport(CoreSession session, String parentPath, File csvFile, String
     List<CSVImportLog> getLastImportLogs(String id, int max, Status... status);
 
     CSVImportResult getImportResult(String id);
+
+    /**
+     * Returns the accepted types for file upload.
+     * 
+     * @since 7.3
+     */
+    Set<String> getAcceptedTypes();
+
+    /**
+     * Prepares an uploaded file, before the actual import.
+     * 
+     * @since 7.3
+     */
+    File prepareUploadedFile(UploadedFile uploadedFile) throws ClientException;
```

Comment:
`org.richfaces.model.UploadedFile` is a high-level UI-related interface that should not appear in low-level services. Please refactor the code a bit (and the accept() method too) to pass just what you need, from what I see the filename, content type and input stream, as distinct arguments.